CMakeLists.txt

```diff
@@ -100,6 +100,13 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# cgreen
+#
+if(NOT CUKE_DISABLE_CGREEN)
+    find_package(cgreen QUIET)
```

Comment:
`QUIET` is a cmake directive, it allows the cmake step to succeed if the `find_package` function fails to find cgreen. Without the QUIET directive cmake can fail there. I'm happy to remove it if you want cmake to fail if the package is not found.